server/routes/overview/prepareTimelineForView.ts

```diff
@@ -1,8 +1,15 @@
 import type { Timeline, TimelineEvent } from 'viewModels'
-import moment from 'moment'
-import dateComparator from './dateComparator'
+import { subSeconds } from 'date-fns'
```

Comment:
There was a lot wrong here!
The original filename was `timelineResolver.ts` with an exported function of `filterTimelineEvents`. It didn't "resolve" anything (FWIW I don't like that term in file/function names - we inherited it from elsewhere); and the exported function didn't filter anything!

What it does do is prepare the `Timeline` (a view model type) for rendering in the nunjucks template.
Our UI designs and requirements for the timeline have some very specific requirements, especially with multiple goals that were all created at the same time